gpkit/interactive/sensitivity_map.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+from gpkit import Monomial
+from gpkit.small_scripts import mag, latex_num
+from gpkit.nomials import MonoEQConstraint
+
+
+BLUE = np.array([16, 131, 246])/255.0
+RED = np.array([211, 24, 16])/255.0
+GRAY = 0.7*np.ones(3)
+
+
+def colorfn_gen(scale, power=0.66):
+    scale = float(scale)
+
+    def colorfn(senss):
+        if senss < 0:
+            senss = -senss
+            color = BLUE
+        else:
+            color = RED
+        blended_color = GRAY + (color-GRAY)*(senss/scale)**power
+        return "[rgb]{%.2f,%.2f,%.2f}" % tuple(blended_color)
+    return colorfn
+
+
+def signomial_print(sig, sol, colorfn, paintby="variables", idx=None):
+    "For pretty printing with Sympy"
```

Comment:
Very much yes. One option would be to build colouring functionality into Signomial, but I wasn't comfortable with that; and this was partly an exercise into how hard it would be for third parties to extend gpkit.plots